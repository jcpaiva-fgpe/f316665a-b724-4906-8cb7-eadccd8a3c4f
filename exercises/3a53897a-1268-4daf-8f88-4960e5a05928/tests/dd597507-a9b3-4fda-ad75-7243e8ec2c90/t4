-4000 20000 -3000 26000
1
-5000 -5000
