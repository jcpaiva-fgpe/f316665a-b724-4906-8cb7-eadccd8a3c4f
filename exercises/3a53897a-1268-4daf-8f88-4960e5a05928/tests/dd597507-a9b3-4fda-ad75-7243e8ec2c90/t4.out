3
Funchal 1
Lisboa 1
Porto 1
